src/modules.js/delete.js

```diff
@@ -0,0 +1,10 @@
+import { saveDb } from './saveDb.js';
+
+export const deleteItem = (index) => {
+  const db = JSON.parse(localStorage.getItem('to-do-list'));
+  localStorage.setItem('to-do-list', JSON.stringify(db));
+  db.splice(index, 1);
+  saveDb(db);
```

Comment:
- [ ] As part of the requirements, Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. currently the indexes are not updated after deleting an item , kindly fix that

Reply:
done